topbeat/etc/fields.yml

```diff
@@ -8,6 +8,90 @@
         Set to "system" to indicate that the statistics are system-wide.
       required: true
 
+- key: core
+  title: "CPU statistics per core"
+  description: >
+    Contains CPU statistics for each core. These statistics are the details that you can get by running the *top* command on Unix systems.
+  fields:
+    - name: id
+      type: long
+      description: >
+       The id of the core X.
+
+    - name: user
+      type: long
+      description: >
+       The amount of CPU time spent in user space on core X.
```

Comment:
Not a windows expert. This how you get the ticks on windows? https://msdn.microsoft.com/en-us/library/system.timespan.tickspersecond(v=vs.110).aspx

Seems to be 10ns (if I didn't miss-count the zeros).

Makes it hardly possible to compare raw tick values between linux and windows systems?